Can't build on OSX Sierra 10.12.6

```
./autogen.sh
./configure
make
```
then getting
```
/usr/include/mach-o/dyld.h:230:13: note: '_dyld_present' has been explicitly marked deprecated here
extern bool _dyld_present(void)                                                              __OSX_AVAILABLE_BUT_DEPRECATED(__MAC_10_1,__MAC_10_5,__IPHONE_NA,__IPHONE_NA);
            ^
1 warning generated.
mv -f benchmark/.deps/unwind_bench-unwind_bench.Tpo benchmark/.deps/unwind_bench-unwind_bench.Po
g++ -DHAVE_CONFIG_H -I. -I./src  -I./src   -D_THREAD_SAFE  -Wall -Wwrite-strings -Woverloaded-virtual -Wno-sign-compare -fno-builtin-malloc -fno-builtin-free -fno-builtin-realloc -fno-builtin-calloc -fno-builtin-cfree -fno-builtin-memalign -fno-builtin-posix_memalign -fno-builtin-valloc -fno-builtin-pvalloc  -Wno-unused-result -fsized-deallocation   -fno-builtin -g -O2 -MT benchmark/unwind_bench-getcontext_light.o -MD -MP -MF benchmark/.deps/unwind_bench-getcontext_light.Tpo -c -o benchmark/unwind_bench-getcontext_light.o `test -f 'benchmark/getcontext_light.cc' || echo './'`benchmark/getcontext_light.cc
In file included from benchmark/getcontext_light.cc:1:
/usr/include/ucontext.h:43:2: error: The deprecated ucontext routines require _XOPEN_SOURCE to be defined
#error The deprecated ucontext routines require _XOPEN_SOURCE to be defined
 ^
benchmark/getcontext_light.cc:4:34: error: unknown type name 'ucontext_t'
extern "C" void getcontext_light(ucontext_t *ctx);
                                 ^
benchmark/getcontext_light.cc:56:34: error: unknown type name 'ucontext_t'
extern "C" void getcontext_light(ucontext_t *ctx) {
                                 ^
3 errors generated.
make: *** [benchmark/unwind_bench-getcontext_light.o] Error 1
```

Xcode 8.3.3 is installed

```
gcc -v
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 8.1.0 (clang-802.0.42)
Target: x86_64-apple-darwin16.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -881,11 +881,13 @@ malloc_bench_shared_full_CXXFLAGS = $(PTHREAD_CFLAGS) $(AM_CXXFLAGS) $(NO_BUILTI
 malloc_bench_shared_full_LDFLAGS = $(PTHREAD_CFLAGS) $(TCMALLOC_FLAGS)
 malloc_bench_shared_full_LDADD = librun_benchmark.la libtcmalloc.la $(PTHREAD_LIBS)
 
+if !OSX
 noinst_PROGRAMS += unwind_bench
 unwind_bench_SOURCES = benchmark/unwind_bench.cc benchmark/getcontext_light.cc
 unwind_bench_CXXFLAGS = $(PTHREAD_CFLAGS) $(AM_CXXFLAGS) $(NO_BUILTIN_CXXFLAGS)
 unwind_bench_LDFLAGS = $(PTHREAD_CFLAGS) $(TCMALLOC_FLAGS)
 unwind_bench_LDADD = librun_benchmark.la libtcmalloc.la $(PTHREAD_LIBS)
+endif !OSX
 
 endif WITH_HEAP_PROFILER_OR_CHECKER
 

